app/api/orders.py

```diff
@@ -68,11 +68,8 @@ def is_payment_valid(order, mode):
         return (order.paid_via == 'paypal') and order.transaction_id
 
 
-def check_billing_info(data):
-    if data.get('amount') and data.get('amount') > 0 and not data.get('is_billing_enabled'):
```

Comment:
Why is amount check removed?

Reply:
I removed this check as now organizer can decide the billing info . Regarding amount check I will restore it .